packages/beasties/src/index.ts

```diff
@@ -26,8 +26,7 @@ import { applyMarkedSelectors, markOnly, parseStylesheet, serializeStylesheet, v
 import { createDocument, serializeDocument } from './dom'
 import { createLogger, isSubpath } from './util'
 
-const removePseudoElementsPattern = /(?<!\\)::?[a-z-]+(?![a-z-(])/gi
-const cleanEmptyNotPattern = /::?not\(\s*\)/g
```

Comment:
do we not need to handle an empty `::not()`?

Reply:
It's also handled by the updated RegExp

```
/(?<!\\)::?[a-z-]+(?:\(.+\))?/gi.test(`p:not(.irrelevant)`)
Outputs: true
```

Reply:
no... curious we were handling it before - maybe it might end up empty based on stripping of unused selectors?

Reply:
Stripping happens at a later stage, and I do not see any failing tests 🤷‍♂️